### PR TITLE
Dark-mode-UI-Fixes

### PR DESCRIPTION
**Pull request by VAIBHAV-25**

### CODE REVIEW

File: src/ui-components/BalanceInput.tsx [hunk 1]
```diff
@@ -134,16 +132,7 @@ const BalanceInput = ({
 			>
 				<Input
 					onBlur={() => onBlur?.()}
-					addonAfter={
-						<div className='flex items-center justify-center gap-1'>
```
Comment (KanishkaRajputd) [inline src/ui-components/BalanceInput.tsx:138]: why we remove this

Comment (KanishkaRajputd) [reply src/ui-components/BalanceInput.tsx:138]: image
icon kha h isme

**Review by KanishkaRajputd — CHANGES_REQUESTED** [review]

File: src/ui-components/BalanceInput.tsx (hunk 1 above)
Comment (VAIBHAV-25) [reply src/ui-components/BalanceInput.tsx:138]: design mai icon ni hai isliye hta diya

Comment (KanishkaRajputd) [reply src/ui-components/BalanceInput.tsx:138]: icon h hato mat

**Review by KanishkaRajputd — APPROVED** [review]

**Pull request merged**